controllers/webhooks/networking/cfroute_validation.go

```diff
@@ -25,6 +26,19 @@ import (
 const (
 	RouteEntityType = "route"
 
+	RouteDecodingError                     = "RouteDecodingError"
```

Comment:
I'd add a `Type` prefix to these constants, since they're not actually errors but the type on them. This also works well with the Message suffix you've given other constants.